src/NzbDrone.Core/Indexers/HttpIndexerBase.cs

```diff
@@ -519,6 +527,11 @@ protected virtual async Task<IndexerResponse> FetchIndexerResponse(IndexerReques
                 {
                     throw new TooManyRequestsException(request.HttpRequest, response);
                 }
+
```

Comment:
Future considerations. Expand this catching to Notifications and client connections as well